PyInstaller/hooks/hook-numpy.py

```diff
@@ -0,0 +1,15 @@
+import os
+import platform
+from PyInstaller.utils.hooks import collect_submodules, get_module_file_attribute
+
+# if we bundle the testing module, this will cause
+# `scipy` to be pulled in unintentionally but numpy imports
+# numpy.testing at the top level for historical reasons.
+# excludedimports = collect_submodules('numpy.testing')
```

Comment:
Why is this line commented out? Looks reasonable to me.

Reply:
`numpy/__init__.py` imports `numpy.testing` and instantiates a pair of benchmarking objects for historical reasons.